### ISSUE TITLE
Implement keyboard accessibility for navigation links

### ISSUE THREAD

**Issue by HaSistrunk** [RockefellerArchiveCenter/rockarch.org#44]
Top `<nav>` is not accessible for keyboard users for either mobile or desktop. Spacebar and Enter should activate dropdowns and submenus shoue be able to be accessed via arrow keys. Consider using `<button>` element or `role="button"` instead of `<href>` to avoid empty links and implement `<ariahaspopup>` set to `true`. When menu is visible, button has `<aria-expanded>` set to `true` and when menu is hidden it is set to `false`.

**Comment by helrond** [comment]
@HaSistrunk Should the page content navigation be accessible in the same way as the top nav?

**Comment by HaSistrunk** [comment]
Oh yes, good point. For the mobile version with the Open/Close Page Menu button on the page content nav, that should also be keyboard accessible and have the ARIA states for `haspopup` and `aria-expanded`

**Comment by helrond** [comment]
note to self: https://github.com/Heydon/inclusive-menu-button

**Comment by helrond** [comment]
Page content nav 922964a929c341d6d8af45e1506771dc1050d6e5
Top nav accessibility
